src/sensors/scripts/lidar.py

```diff
@@ -11,13 +11,13 @@ class Lidar:
     """
 
     def __init__(self):
-        self.lidar_pub = rospy.Publisher('lidar',Float64 ,queue_size=10)
+        self.lidar_pub = rospy.Publisher('lidar',PointCloud ,queue_size=10)
 
     def send_lidar_data(self):
         rospy.init_node('talker', anonymous=True)
         rate = rospy.Rate(2)
         while not rospy.is_shutdown():
-            self.lidar_pub(PointCloud())
+            self.lidar_pub.publish(PointCloud())
```

Comment:
Any idea what the previous statement did that you removed? The one you added looks correct, whereas the other one did not (was it even doing anything?)

Reply:
nothing would just throw an error, simple mistake that was not caught cause lidar never ran